sample-apps/spark/build.gradle.kts

```diff
@@ -13,6 +13,8 @@ dependencies {
   implementation("org.apache.logging.log4j:log4j-core")
   implementation("software.amazon.awssdk:s3")
   implementation("software.amazon.awssdk:sts")
+  implementation("io.opentelemetry:opentelemetry-sdk-metrics:1.10.0-alpha")
```

Comment:
The sample app includes the javaagent so shouldn't need the SDK nor is that really an allowed configuration. Is something not working right now with the javaagent?

Reply:
The issue is we are not publishing anything to the collector since the api build a no op provider.